frontend/src/AppBuilder/AppCanvas/Grid/Grid.jsx

```diff
@@ -354,6 +355,27 @@ export default function Grid({ gridWidth, currentLayout }) {
     [boxList, currentLayout, gridWidth]
   );
 
+  const handleCanvasClick = useCallback(
+    (e) => {
+      const realCanvas = e.target.closest('.real-canvas');
+      if (realCanvas) {
+        const rect = realCanvas.getBoundingClientRect();
+        const x = e.clientX - rect.left;
+        const y = e.clientY - rect.top;
+        setLastClickPosition({ x, y });
+      }
+    },
+    [setLastClickPosition]
+  );
+
+  useEffect(() => {
```

Comment:
can we add this as onClick attribute int real canvas itself?

Reply:
Yes, added onClick to real-canvas